integrations/gcp/.port/resources/port-app-config.yaml

```diff
@@ -0,0 +1,86 @@
+resources:
+  - kind: cloudresourcemanager.googleapis.com/Organization
+    selector:
+      query: "true"
+    port:
+      entity:
+        mappings:
+          identifier: ".name"
+          blueprint: '"gcpOrganization"'
+          title: ".versioned_resources | max_by(.version).resource | .displayName"
```

Comment:
seems like the `max_by` is being repeated many times in almost all of the mappings, lets check if we can query only the latest version from the API, if not, lets add a comment about it

Reply:
No way to get only the latest version via api. will add comment